### ISSUE TITLE
HTML elements as labels for radio buttons

### ISSUE THREAD

**Issue by stla** [rstudio/shiny#1437]
Hello, 

 It would be nice to have the possibility to put HTML in the labels of the radio buttons. 
Here is the function I wrote myself to do so (and it works fine):

```
radioButtonsHTML <- function(inputId, label, choices, choicesLabels=NULL, selected = NULL, inline = FALSE, width = NULL){
  if(is.null(choicesLabels)){
    return(radioButtons(inputId, label, choices, selected, inline, width))
  }else{
    if(length(choicesLabels) != length(choices)){
      stop("")
    }else{
      rb <- radioButtons(inputId, label, choices, selected, inline, width)
      for(i in seq_along(choices)){
        rb$children[[2]]$children[[1]][[i]]$children[[1]]$children[[2]]$children[[1]] <- HTML(choicesLabels[i])
      }
      return(rb)
    }
  }
}
```

**Comment by bborgesr** [comment]
I'm having trouble understanding what this function does. Could you post a usage example?

**Comment by stla** [comment]
This is the problem as [here](http://stackoverflow.com/questions/32169748/how-can-i-have-html-elements-as-labels-for-radio-buttons). 
Below is a usage example of my function:
```
        radioButtonsHTML("parameter", "Calculate:",
                     c("delta", "sigma", "p"), 
                     choicesLabels = c("&delta;", "&sigma;", "p"))
```
This gives:
![capture du 2016-10-25 14 28 00](https://cloud.githubusercontent.com/assets/4466543/19685980/69af19cc-9abf-11e6-98e7-a69f41c33a72.png)

**Comment by daattali** [comment]
Similarly, I've seen many people ask for a way to have an image as the option (so this would also be solved by allowing any HTML)

**Comment by bborgesr** [comment]
Closed via d62a2fc (#1521)
